elements_of_maps/elements_of_maps.md

```diff
@@ -0,0 +1,408 @@
+
+![An outline of Philadelphia is marked with 28 blue plus signs and 2 orange stars.](media/ped_v_bike_deaths2019.jpg)
+
+Lines can also be represented differently to show, symbolically rather than with color, how they differ. A dotted line might represent one type of boundary or path, while a dashed line could represent another.
+
+![The Broad Street Line is marked as a dotted line while the Market-Frankford Line is marked with a dashed line.](media/Philadelphia_highspeed_trains_linestyle.jpg)
+
+Shapes can also be patterned or marked with symbols to indicate particular features. Be careful, though, as using patterns to differentiate regions can quickly get very busy-looking.
+
+![Philadelphia is partitioned into regions based on the day of the week for municipal trash collection. Regions are marked with left and right hatches, circles, dots, and exes.](media/Phila_trash_pickup.jpg)
+
+Combining Types of Maps
+--------------------
+You are not restricted to using only one type of map at a time. The data you want to display might benefit from a combination of multiple features. Perhaps you want to look at a heat map of pedestrian stops made by police, overlayed with the locations of police stations, schools, and liquor stores, each identified by a different symbol.
+
+Or perhaps a choropleth could show you the concentration of green spaces in the city while each zip code is also labeled with the medium household income of its residents.
+
+Just because you can use different elements to create maps that show many types of data together, doesn't mean it is a good idea. Remember that the best maps communicate their message clearly. If fitting all of your data onto a single map means none of it is understandable, you probably need to use more than one map to display that data!
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Making good maps is hard work and can feel more like art than science. Creating a beautiful map that both communicates the data you are trying to show clearly is a lofty goal. 
+
+Focus on clearly presenting your data and the rest will follow.
+
+</div>
+
+## Reference
+
+Elements of Maps
+--------------
+
+| Shapes       | Colors                           | Sizes                    | Language    | Symbols |
+|:--|:--|:--|:--|:--|
+| Points   |  Hue (water is blue)          | Thick / thin lines   |  Scales   | Specify locations |
+| Lines    |  Intensity (e.g. water depth) | Large / small points |   Numbers | Solid / broken lines |
+| Polygons |                                  |                   |  Words    | Patterned regions |
+
+
+Types of Maps
+--------------
+
+We have seen several named types of maps in this lesson:
+
+
```

Comment:
Modify to make 3 columns, Map Type, Description, Example Image.

Minimal alt text on example image because everything is contained in the text description column.